keps/sig-node/3327-align-by-socket/kep.yaml

```
@@ -28,13 +28,13 @@ latest-milestone: "v1.25"
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.25"
-  beta: "v1.26"
-  stable: "v1.27"
+  beta: "v1.27"
+  stable: "v1.28"
 
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
-  - name: "CPUManagerPolicyAlphaOptions"
+  - name: "CPUManagerPolicyAlphaOptions", "CPUManagerPolicyBetaOptions", "CPUManagerPolicyOptions"
     components:
       - kubelet
```

CI is not happy because we need specify feature gates like below:
```suggestion
  - name: "CPUManagerPolicyOptions"
  - name: "CPUManagerPolicyAlphaOptions"
  - name: "CPUManagerPolicyBetaOptions"
    components:
      - kubelet
```

---

Thanks @swatisehgal! Was trying to figure out right format.
Updated with your suggestion. also ran hack/verify.sh to ensure CI is ok